src/pages/AdvancedSearch/ResultsBlock/ResultsBlock.tsx

```diff
@@ -0,0 +1,84 @@
+import {
+  CodeSnippet,
+  CodeSnippetBlockAppearance,
+  Spinner,
+} from "@canonical/react-components";
+import { useState } from "react";
+
+import {
+  getCrossModelQueryLoaded,
+  getCrossModelQueryLoading,
+  getCrossModelQueryResults,
+} from "store/juju/selectors";
+import { useAppSelector } from "store/store";
+
+enum CodeSnippetView {
+  TREE = "tree",
+  JSON = "json",
+}
+
+const ResultsBlock = (): JSX.Element | null => {
+  const isCrossModelQueryLoading = useAppSelector(getCrossModelQueryLoading);
+  const isCrossModelQueryLoaded = useAppSelector(getCrossModelQueryLoaded);
```

Comment:
I replaced the `isCrossModelQueryRequestMade` state with the loaded state from Redux as I think this will serve the same purpose.